Orange/widgets/regression/owsgdregression.py

```diff
@@ -53,82 +58,58 @@ def __init__(self):
-        )
-
-        parambox = gui.widgetBox(box, orientation="horizontal")
-
-        box = gui.widgetBox(self.controlArea, self.tr("Penalty"))
-        buttonbox = gui.radioButtonsInBox(
-            box, self, "penalty_type",
-            btnLabels=["Absolute norm (L1)",
-                       "Euclidean norm (L2)",
-                       "Elastic Net (both)"],
-            callback=self._on_penalty_changed
-        )
-
-        parambox = gui.widgetBox(box, orientation="horizontal")
-
-        box = gui.widgetBox(self.controlArea, self.tr("Learning rate"))
-        buttonbox = gui.radioButtonsInBox(
-            box, self, "learning_rate",
-            btnLabels=["Inverse scaling",
-                       "Constant"],
-            callback=self._on_lrate_changed
-        )
-
-
-
-        box = gui.widgetBox(self.controlArea, self.tr("Constants"))
-
-        form = QtGui.QFormLayout()
-        form.setContentsMargins(0, 0, 0, 0)
-
-        box.layout().addLayout(form)
-
-        alpha = gui.doubleSpin(box, self, "alpha", 0.0, 10.0, step=0.0001)
-        form.addRow("Alpha:", alpha)
-
-        spin = gui.doubleSpin(box, self, "eta0", 0.0, 10, step=0.01)
-        form.addRow("Eta0:", spin)
-
-        epsilon = gui.doubleSpin(box, self, "epsilon", 0.0, 10.0, step=0.01)
-        form.addRow("Epsilon:", epsilon)
-
-        l1_ratio = gui.doubleSpin(box, self, "l1_ratio", 0.0, 10.0, step=0.01)
+        def add_form(box):
+            gui.separator(box)
+            box2 = gui.hBox(box)
+            gui.rubber(box2)
+            form = QtGui.QFormLayout()
+            form.setContentsMargins(0, 0, 0, 0)
+            box2.layout().addLayout(form)
+            return form
+
+        box = gui.radioButtonsInBox(
+            self.controlArea, self, "loss_function", box="Loss function",
+            btnLabels=self.LOSS_FUNCTIONS, callback=self._on_func_changed)
+        form = add_form(box)
+        epsilon = gui.doubleSpin(
+            box, self, "epsilon", 0.0, 10.0, 0.01, controlWidth=70)
+        form.addRow("ε:", epsilon)
+
+        box = gui.radioButtons(
+            self.controlArea, self, "penalty_type", box="Penalty",
+            btnLabels=self.PENALTIES, callback=self._on_penalty_changed)
+        form = add_form(box)
+        alpha = gui.doubleSpin(
+            box, self, "alpha", 0.0, 10.0, 0.0001, controlWidth=70)
+        form.addRow("α:", alpha)
+        l1_ratio = gui.doubleSpin(
+            box, self, "l1_ratio", 0.0, 10.0, 0.01, controlWidth=70)
         form.addRow("L1 ratio:", l1_ratio)
 
-        power_t = gui.doubleSpin(box, self, "power_t", 0.0, 10.0, step=0.01)
+        box = gui.radioButtonsInBox(
+            self.controlArea, self, "learning_rate", box="Learning rate",
+            btnLabels=self.LEARNING_RATES, callback=self._on_lrate_changed)
+        form = add_form(box)
+        spin = gui.doubleSpin(
+            box, self, "eta0", 0.0, 10, 0.01, controlWidth=70)
+        form.addRow("η0:", spin)
```

Comment:
Yes. I was lazy. @wadhwasahil, can you enclose the 0 in `<sub>`, </sub>`? And do the same in the report?